src/mmw/mmw/settings/layer_settings.py

```diff
@@ -119,6 +128,16 @@
         'minZoom': 10
     },
     {
+        'code': 'drb_streams',
+        'display': 'DRB Stream Network',
+        'table_name': 'drb_streams_50',
+        'boundary': False,
+        'stream': True,
+        'overlay': True,
+        'perimeter': drb_perimeter # Layer is only selectable when viewport
```

Comment:
Sorry, with more context.  I think we should add a `'minZoom': 11` to this block.